src/ui/public/embeddable/get_index_pattern.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import chrome from 'ui/chrome';
+import { StaticIndexPattern } from 'ui/index_patterns';
+import { getFromSavedObject } from 'ui/index_patterns/static_utils';
+import { VisSavedObject } from 'ui/visualize/loader/types';
+
+export async function getIndexPattern(
```

Comment:
What was the reasoning behind putting this function in the embeddable folder?  It doesn't seem to be tied to embeddables, and relies on specific visualization saved object knowledge.

Reply:
You're right, this should be moved to public/visualize somewhere

Reply:
Do you mind pulling together a follow up PR for this move?